src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -35,45 +35,41 @@ public sealed class SonarQubeIssuesProvider : ISonarQubeIssuesProvider, IDisposa
         private readonly TimeSpan MillisecondsToWaitForInitialFetch = TimeSpan.FromMinutes(1);
         private readonly Task initialFetch;
 
-        private readonly ITimer refreshTimer;
-        private readonly IActiveSolutionBoundTracker solutionBoundTracker;
         private readonly ISonarQubeService sonarQubeService;
+        private readonly string sonarQubeProjectKey;
+        private readonly ITimer refreshTimer;
 
         private IList<SonarQubeIssue> cachedSuppressedIssues;
         private bool isDisposed;
         private CancellationTokenSource cancellationTokenSource;
 
-        public SonarQubeIssuesProvider(ISonarQubeService sonarQubeService, 
-            IActiveSolutionBoundTracker solutionBoundTracker,
+        public SonarQubeIssuesProvider(ISonarQubeService sonarQubeService,
+            string sonarQubeProjectKey,
```

Comment:
We still need to know the SQ project id, which we previously got from the solution tracker

Reply:
This probably fits on the previous line.

Reply:
I'll look at it when refactoring the constructor.